### ISSUE TITLE
when run seedrecover.py,  many noise _create_built_program_from_source_cached warnings.

### ISSUE THREAD

**Issue by CryptoCoinLab** [3rdIteration/btcrecover#467]
python seedrecover.py   --enable-opencl --mnemonic "emit emit......."  --no-dupchecks   --addr-limit 2 --addressdb E:\addresses-btc.db  --bip32-path m/44'/0'/0'/0 m/44'/0'/0' m/0' m/0 m/0'/0' m/0'/0 --dsw --wallet-type bip39 --language en --mnemonic-length 12 --no-eta

pyopencl : 2024.1
NVIDIA CUDA 12.4
NVIDIA GeForce RTX 3060 Laptop GPU Driver: 31.0.15.5222

many noise warnings:

_create_built_program_from_source_cached(
C:\Python310\lib\site-packages\pyopencl\cache.py:495: CompilerWarning: Built kernel retrieved from cache. Original from-source build had warnings:
Build on <pyopencl.Device 'NVIDIA GeForce RTX 3060 Laptop GPU' on 'NVIDIA CUDA' at 0x20037ea2580> succeeded, but said:

<kernel>:628:20: warning: initializing '__global unsigned long *' with an expression of type 'unsigned long const __global[4]' discards qualifiers
    __global word *pwdBuffer = pwdbuffer_arg[0].buffer;
                   ^           ~~~~~~~~~~~~~~~~~~~~~~~
(): Warning: Function hash_main is a kernel, so overriding noinline attribute. The function may be inlined when called.
(): Warning: Function pbkdf2 is a kernel, so overriding noinline attribute. The function may be inlined when called.
(): Warning: Function hmac_main is a kernel, so overriding noinline attribute. The function may be inlined when called.
(): Warning: Function pbkdf2_saltlist is a kernel, so overriding noinline attribute. The function may be inlined when called.

**Comment by 3rdIteration** [comment]
This is normal with the latest Nvidia drivers, everything still works as expected though

**Comment by CryptoCoinLab** [comment]
Yeah, you are right ,  but how to get ride of these  noise warnings ?
